calcom/cal.com
# Add NEXTAUTH_URL to app.json to address heroku deploy failure

## What does this PR do?
Fixes #2936 
Now issues `2022-06-17T06:26:22.137118+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=foolio.herokuapp.com request_id=bc591b56-0605-4432-a162-b4465c8d0f0d fwd="73.93.110.91" dyno= connect= service= status=503 bytes= protocol=https
2022-06-17T06:26:23.320175+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=foolio.herokuapp.com request_id=1df5d73e-cb78-4faa-9cba-b2d8011fa738 fwd="73.93.110.91" dyno= connect= service= status=503 bytes= protocol=https` error when visiting site, which is a different issue, but is not a regression.

## Type of change
- [ X] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
Visit https://dashboard.heroku.com/new?template=https%3A%2F%2Fgithub.com%2Fchoongkyu%2Fcalendso%2Ftree%2Fheroku-url and add config vars and click deploy

Confirm that success message like the bottom appears in the log and build completes with no errors:
```
-----> Caching build
       - yarn cache
       
-----> Build succeeded!
 !     Unmet dependencies don't fail yarn install but may cause runtime issues
       https://github.com/npm/npm/issues/7494
-----> Discovering process types
       Procfile declares types -> web
-----> Compressing...
       Done: 284.6M
-----> Launching...
       Released v6
       https://foolio.herokuapp.com/ deployed to Heroku
```

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works (n/a)
